components/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from "react"
+import Header from "./Header"
+
+const Layout: React.FC<{children: ReactNode}> = (props) => {
+  return (
+    <>
```

Comment:
nit: delete extraneous brackets

Reply:
just kidding, we needed these outside brackets to wrap around multiple elements